### DISCUSSION TITLE
Redlining - Objekt auswählen-Hinweis - Rechtschreibfehler

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#55] [General]
Hallo Jürgen!

Eine Userin hat uns darauf hingewiesen, dass beim Redlining beim Hinweis ein Rechtschreibfehler drin ist. Könntest du den bitte im Quellcode ausbessern, damit ich ihn nicht bei jedem Softwareupdate wieder korrigieren muss. Danke!

![image](https://user-images.githubusercontent.com/122986367/232465829-97f3dfc6-e760-47cd-ad0f-71f8a62b6d9a.png)

> hier ist sicher "klicken" gemeint statt "klichen" (s. gelb hinterlegte Box auf Screenshot)

Lg, Andi

**Comment by jugstalt** [comment]
Danke für den Hinweis. Ist im nächsten Build behoben.
